Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfiguration.mm

```diff
@@ -128,6 +128,7 @@ @implementation WKWebViewConfiguration {
     LazyInitialized<RetainPtr<WKWebsiteDataStore>> _websiteDataStore;
     LazyInitialized<RetainPtr<WKWebpagePreferences>> _defaultWebpagePreferences;
     WeakObjCPtr<WKWebView> _relatedWebView;
+    WeakObjCPtr<WKWebView> _relatedWebViewForStorage;
```

Comment:
I am fine with this but I think we may be able to come up with a clearer name, e.g. `webViewToCloneSessionStorageFrom`, or similar.